android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -75,7 +77,7 @@ protected void onResume() {
         int spanCount = (int) (width/250.00);
 
         sessionRecyclerView.setHasFixedSize(true);
-        final GridLayoutManager gridLayoutManager = new GridLayoutManager(this,spanCount);
+        gridLayoutManager = new GridLayoutManager(this,spanCount);
```

Comment:
@harshitagupta30 Sorry, I didn't get you here. Do you want me to change this line 
`gridLayoutManager = new GridLayoutManager(this,spanCount);`  to 
`gridLayoutManager = new GridLayoutManager(this, spanCount);`